src/index.ts

```diff
@@ -1916,103 +1854,6 @@ async function claimLinkGasless({
 	}
 }
```

Comment:
You deleted `claimLinkXChainGasless`, but integrators (e.g. the peanut UI) still need a way to to claim a link xchain. This shouldn't be deleted, but just modified to use the new code and call api endpoint

(or is it deleted because new API xchain endpoint isn't ready yet? in that case keep the function but empty the contents and mark it as deprecated)

Reply:
> (or is it deleted because new API xchain endpoint isn't ready yet? in that case keep the function but empty the contents and mark it as deprecated)

yuuuuuup

Done!